### PR TITLE
Update object-visibility.md

### PR DESCRIPTION
**Pull request by gregorypierce**
There is no version 6.0.0 of MLAPI

**Select the type of change:**

- [ x] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

Fixed issue with MLAPI version listed.

**Issue Number:**

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=231) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=231) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=231) it.</sub>

**Comment by Briancoughlin** [comment]
there is a version 6.0 of MLAPI its a pre unity version. I will try and see if i can find a way to cover this in the docs beyond the intro pages.

File: docs/mlapi-basics/object-visibility.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Object Visibility
 sidebar_label: Object Visibility
 ---
 
-Starting with MLAPI version 6.0.0, clients have no explicit knowledge of all objects or clients that are connected to the server.
+Starting with MLAPI version 0.1.0, clients have no explicit knowledge of all objects or clients that are connected to the server.
```
Comment (LukeStampfli) [inline docs/mlapi-basics/object-visibility.md:7]: This is actually not correct. Object Visibility was introduced in version 6.0.0 and not 0.1.0.

The thing is we started fresh from the 0.1.0 version when we acquired MLAPI. We will probably need to update/remove comments like this which reference an old and no longer relevant version of MLAPI.

**Comment by Briancoughlin** [comment]
refers to pre acquisition version of MLAPI by Unity. Closing as the text is correct

**Pull request closed without merging**